test/runtests.jl

```diff
@@ -1,4 +1,4 @@
-using Fairness
+using Fairness2
```

Comment:
The name of the package should be Fairness and not Fairness2. This is causing the tests to fail

Reply:
Hi!
I Have changed the package name and it is working fine now.
Also added comments to the code.